openfold/data/data_pipeline.py

```diff
@@ -784,6 +784,45 @@ def read_template(start, size):
 
         return all_hits
 
+    def _parse_template_hits(
```

Comment:
Should also be gone after rebase

Reply:
Ah sorry this file is outdated. I will update data_pipeline.py with the version on multimer branch now and push it.